packages/@okta/vuepress-site/docs/guides/quickstart/using-console/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Using the Console
+---
+
+The Developer Console is the web UI that provides you with a window into your org and lets you configure and manage it. The URL to access your Developer Console consists of your Okta domain plus `-admin`, e.g., `https://dev-l33337.okta.com`.
+
+#### Dashboard
+
+On signing in to Developer Console, you see the Dashboard, a landing page that provides a summary of activity in your org.
```

Comment:
Should we start by describing the `Get Started` page, which is the landing page for developers signing in to their org for the first time?

Reply:
@alexdahl-okta, the initial wizard was mentioned at the end of the previous page:

"A wizard launches to help you through basic setup for a few commons scenarios. You can choose to exit the wizard by clicking **Dashboard**. (In the next sections of this QuickStart, we won’t use the wizard, but rather the regular Dashboard menus that can be used on an ongoing basis.)"